cn_indian_payroll/cn_indian_payroll/report/epf_challan_report/epf_challan_report.py

```
@@ -15,39 +16,55 @@ def get_salary_slips(filters=None):
+            if earning.salary_component == basic_component:
+                basic = earning.amount
+            elif earning.salary_component == da_component:
+                da = earning.amount
+
+        # Calculate EPF values
+        epf_value = sum(
+            d.amount for d in each_salary_slip.get("deductions", [])
+            if frappe.get_value("Salary Component", d.salary_component, "component_type") == "EPF"
+        )
+        
+        epf_value_employer = sum(
```

Ruff (0.8.2)

Local variable `epf_value_employer` is assigned to but never used

Remove assignment to unused variable `epf_value_employer`

(F841)